packages/components/src/LikertScaleLegacy/_docs/LikertScaleLegacy.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { LikertScaleLegacy } from "../index"
+import { Scale, ScaleItem } from "../types"
+
+const scale: Scale = [
+  {
+    value: -1,
+    label: "Not rated",
+  },
+  {
+    value: 1,
+    label: "Strong Disagree",
+  },
+  {
+    value: 2,
+    label: "Disagree",
+  },
+  {
+    value: 3,
+    label: "Neither agree or disagree",
+  },
+  {
+    value: 4,
+    label: "Agree",
+  },
+  {
+    value: 5,
+    label: "Strongly agree",
+  },
+]
+
+const meta = {
+  title: "Components/LikertScaleLegacy",
+  component: LikertScaleLegacy,
+  args: {
+    scale,
+    labelId: "labelId",
+    selectedItem: scale[0],
+    onSelect: (): void => undefined,
+  },
+} satisfies Meta<typeof LikertScaleLegacy>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  render: args => {
+    const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
```

Comment:
Debating whether or not I should make the Playground work...

Reply:
The story seems to work on the mdx page 😄 or do you mean show we not have it work because its a legacy likert scale?

Reply:
I made it work, but I'm debating removing the functionality in the example cause it might imply no effort on the consumer side to make it work in their code base...

Reply:
Ah gotcha. Couldn't we update the canvas src to showcase the mimimum code required to get that story up and running?

Reply:
Hm I guess there's not much to toggle in controls, so may as well hardcode it

Reply:
Nice 👍 I can't imagine it changing so I think that should be good